fulp_modules/main_features/bloodsuckers/code/bloodsucker/bloodsucker_crypt.dm

```diff
@@ -602,6 +613,100 @@
+			)
+		var/revive_response = show_radial_menu(user, src, revive_options, radius = 36, require_near = TRUE)
+		switch(revive_response)
+			if("Yes")
+				to_chat(user, "<span class='danger'>You have brought [target] back from the dead!</span>")
+				to_chat(target, "<span class='announce'>As blood drips over your body, your heart begins to beat... You live again!</span>")
+				user.blood_volume -= 150
+				target.revive(full_heal = TRUE, admin_revive = TRUE)
+				ADD_TRAIT(target, TRAIT_MUTE, BLOODSUCKER_TRAIT)
+				target.become_husk()
+				return
+			else
+				to_chat(user, "<span class='danger'>You decide not to revive [target].</span>")
+				unbuckle_mob(target)
+				useLock = FALSE
+				return
+
+	var/static/list/monster_choices = list(
+		TZIMISCE_HUSK,
+		TZIMISCE_TWOARMED,
+		TZIMISCE_CLAWMONSTER,
+		TZIMISCE_TRIPLECHESTED,
+	)
+	var/list/options = list()
+	options = monster_choices
+	var/answer = tgui_input_list(user, "We have the chance to shapeshift our victim into something greater, how should we mutilate their corpse?", "What do we do with our victim?", options)
+
+	var/make_monster = TRUE /// Are we making them into a monster?
+	var/mob/living/simple_animal/monster /// The monster the target is being transferred to upon succesful ritual.
+	var/blood_lost /// The amount of blood lost by the Tzimisce upon succesfully shapeshifting the target.
+	var/blood_gained /// The amount of blood gained by the Tzimisce upon a successful shapeshift.
+	var/lose_items = TRUE /// Do we lose items upon succesful rituals? Only FALSE for husks as they do not become simplemobs.
+	var/user_message = "You have shaped [target] into a two-armed monster!" /// The message that the Tzimisce gets upon a successful ritual.
+	var/target_message = "You've been turned into a monster!" /// The message that the victim gets upon a succesful ritual.
+
+	if(!answer)
+		to_chat(user, "<span class='notice'>You decide to leave your victim just the way they are.</span>")
+		return
+
+	switch(answer)
+		if(TZIMISCE_HUSK)
+			blood_gained = 200
+			lose_items = FALSE
+			make_monster = FALSE
+			user_message = "You grotesquely shape [target]'s body, turning [target.p_them()] into a Living Husk!"
+			target_message =  "You've been turned into a Living Husk!"
+			ADD_TRAIT(target, TRAIT_MUTE, BLOODSUCKER_TRAIT)
+			target.become_husk()
+		if(TZIMISCE_TWOARMED)
+			blood_lost = 100
+			monster = /mob/living/simple_animal/hostile/retaliate/tzimisce/twoarmed
+		if(TZIMISCE_CLAWMONSTER)
+			blood_lost = 250
+			monster = /mob/living/simple_animal/hostile/retaliate/tzimisce/clawmonster
+			user_message = "You have shaped [target] into a gluttonous, clawed monster!"
+		if(TZIMISCE_TRIPLECHESTED)
+			blood_lost = 300
+			monster = /mob/living/simple_animal/hostile/retaliate/tzimisce/triplechested
+			user_message = "You have shaped [target] into a triple-chested, bulky monster!"
+
+	INVOKE_ASYNC(target, /mob.proc/emote, "scream")
+	if(!do_mob(user, src, 20 SECONDS))
```

Comment:
ah yes, standing next to a rack for 20 seconds while you wait for a meter to fill up, truly the pinnacle of fun and engaging gameplay